src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  height: 80vmin;
+  width: 80vmin;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
```

Comment:
It's a good practice to include a `max-width` property for large fixed-size elements to ensure that the layout remains responsive on smaller screens. Consider adding a `max-width` to `.stopwatch` to improve responsiveness.